beater/config/sampling.go

```diff
@@ -143,5 +145,7 @@ func defaultTailSamplingConfig() TailSamplingConfig {
 		IngestRateDecayFactor: 0.25,
 		StorageGCInterval:     5 * time.Minute,
 		TTL:                   30 * time.Minute,
+		StorageLimit:          3 * 1024 * 1024 * 1024, // 3GB default size limit?
```

Comment:
I am unsure whether we should default this size to unbounded or to something like 3 or 4GB, which seems like a decent amount of storage to default to. This is kind of an arbitrary threshold.